packages/lesswrong/components/revisions/TagPageRevisionSelect.tsx

```diff
@@ -36,7 +37,7 @@ const TagPageRevisionSelect = ({ classes }: {
   
   const compareRevs = useCallback(({before,after}: {before: RevisionMetadata, after: RevisionMetadata}) => {
     if (!tag) return;
-    navigate(`/compare/${tagUrlBase}/${tag.slug}?before=${before.version}&after=${after.version}`);
+    navigate(`/compare/${tagUrlBaseSetting.get()}/${tag.slug}?before=${before.version}&after=${after.version}`);
```

Comment:
Broken link. The routes table still says `/compare/tag` and needs to be updated so this will point at it (and have a redirect at the old path).